src/workspacer.FocusBorder/IFormProxy.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+
+namespace workspacer.FocusBorder
+{
+    public interface IFormProxy<Form>
```

Comment:
In C# when having type-arguments, it's customary to prefix them with `T` to avoid confusion inside the interface or class which depends on them.

`TForm` would be better here. Would it also make sense to apply any constraints to what can be a form in this context?